apps/web/pages/welcome.tsx

```diff
@@ -35,6 +39,27 @@ export default function Welcome(props) {
         setStage(stage + 1);
     }
 
+    useEffect(() => {
+        const hrefs = window.location.href.split('/');
+        const len = hrefs[3];
+      
+      switch (len){
+        case "de":
+          messages = de;
+          break;
+        default:
+          messages = en;
```

Comment:
```suggestion
          // no need to override messages, since en is the default
```
I think we can just add a comment here, since the en translation is the default.